src/javascript/crypto/e2e/otr/session.js

```diff
@@ -105,10 +105,8 @@ e2e.otr.Session.prototype.getAuthState = function() { return this.authState_; };
  * @param {!e2e.otr.constants.AUTHSTATE} nextState The new auth state.
  */
 e2e.otr.Session.prototype.setAuthState = function(nextState) {
-  if (nextState) {
```

Comment:
I'm not even sure why I had this check in the first place since the parameter is defined as non-nullable and authstate doesn't allow undefined